src/m_argv.c

```diff
@@ -417,3 +417,16 @@ const char *M_GetExecutableName(void)
     return M_BaseName(myargv[0]);
 }
 
+const char *exedir;
```

Comment:
This should not be const since it's been allocated from the heap. It can't be freed (if someone wants to do that) without casting the const away.